src/ReceivingList/utils.js

```diff
@@ -76,7 +76,7 @@ export const buildTitlesQuery = (queryParams) => {
 
   const queryParamsFilterQuery = buildFilterQuery(queryParams, (query, qindex) => {
     if (qindex) {
-      return `(${qindex}=${query}*)`;
+      return `(${qindex}==*${query}*)`;
```

Comment:
isn't it slower and will give partial matches?

Reply:
as I understand it should give, POLs work in this way an AM expects the same